{{cookiecutter.bot_project_name}}/Dockerfile

```diff
@@ -1,15 +1,25 @@
 FROM python:3.9-slim
 
+EXPOSE 8000
+
 ENV PYTHONUNBUFFERED 1
 ENV UVICORN_CMD_ARGS ""
 
-ARG NON_ROOT_USER=express_bot
-RUN useradd --create-home ${NON_ROOT_USER}
+ENV APP_USER=appuser
+ENV VIRTUAL_ENV=/home/$APP_USER/.venv/bin
+ENV USER_EXECUTABLES=/home/$APP_USER/.local/bin
```

Comment:
I suggest rename VIRTUAL_ENV to VENV_PATH (or mb PATH_VENV) and USER_EXECUTABLES to USER_PATH (or PATH_USER)

Reply:
Changed